README.md

```diff
@@ -30,7 +30,6 @@ Things you may want to cover:
   - [🛠 Built With](#built-with)
```

Comment:
![image](https://github.com/rahimialijan/hello-rails-react/assets/93039359/b2af2c95-309e-4f08-ae51-4521f667f228)

**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you remove this information highlighted in the image above since it is only for the person who is making his readme file and now you have made it so there is no need of this information 🤔 Kindly remove it because it is making your readme file not clean and professional 😨